docs/getting-started/app-custom-theme.md

```diff
@@ -254,71 +254,7 @@ const LogoFull = () => {
 };
 ```
 
-## Custom Homepage
+## Custom Home Page
 
 In addition to a custom theme, a custom logo, you can also customize the
-homepage of your app. To do that we need to go through a few steps.
-
-### Setting up the Home Page
-
-1. Create a Home Page Component that will be used for composition.
-
-`packages/app/src/components/home/HomePage.tsx`
-
-```tsx
-import React from 'react';
-
-export const HomePage = () => {
-  return {
-    /* TODO: Compose a Home Page here */
-  };
-};
-```
-
-2. Add a route where the homepage will live, presumably `/`.
-
-`packages/app/src/App.tsx`
-
-```tsx
-import { HomepageCompositionRoot } from '@backstage/plugin-home';
-import { HomePage } from './components/home/HomePage';
-
-// ...
-<Route path="/" element={<HomepageCompositionRoot />}>
-  <HomePage />
-</Route>;
-// ...
-```
-
-### Composing your Home Page
-
-Composing a Home Page is no different from creating a regular React Component,
-i.e. the App Integrator is free to include whatever content they like. However,
-there are components developed with the Home Page in mind. If you are looking
-for components to use when composing your homepage, you can take a look at the
-[collection of Homepage components](https://backstage.io/?path=/story/plugins-home-components)
-in storybook. If you don't find a component that suits your needs but want to
-contribute, check the
-[Contributing documentation](https://github.com/backstage/backstage/blob/master/plugins/home/README.md#contributing).
-
-> If you want to use one of the available homepage templates you can find the
-> [templates](https://backstage.io/storybook/?path=/story/plugins-home-templates)
-> in the storybook under the "Home" plugin. And if you would like to contribute
-> a template, please see the
-> [Contributing documentation](https://github.com/backstage/backstage/blob/master/plugins/home/README.md#contributing)
-
-```tsx
-import React from 'react';
-import Grid from '@material-ui/core/Grid';
-import { HomePageCompanyLogo } from '@backstage/plugin-home';
-
-export const HomePage = () => {
-  return (
-    <Grid container spacing={3}>
-      <Grid item xs={12} md={4}>
-        <HomePageCompanyLogo className={container} />
-      </Grid>
-    </Grid>
-  );
-};
-```
+homepage of your app. Read the full guide on the [next page](homepage.md).
```

Comment:
Still some inconsistencies of "homepage" vs "home page" :) This seems to be a pretty contended topic! https://english.stackexchange.com/questions/112038/home-page-or-homepage

Reply:
Good point @freben when writing the blog post I think we said.

"Home" is the name of the plugin, use that for when referring to the plugin itself.
"homepage" when talking about customising your homepage.
"Homepage Templates" as a product name for the actual templates.

looping in @fengypants here too!

Reply:
Yep, lotsa spellings! (I think there is also camel case "HomePage" out there.) We decided on "homepage" (one word) and capitalized the way Emma outlined above. 

For example: "As part of the Home plugin for Backstage, you'll find Backstage Homepage Templates, which let you easily customize the look and content of your developer portal's homepage." 

Terrible sentence, but hopefully gets the idea across.

Reply:
homepage it is!